gen3-integration-tests/tests/test_dataguids.py

```diff
@@ -0,0 +1,94 @@
+import os
+import pytest
+import requests
+import json
+
+from utils import logger, TEST_DATA_PATH_OBJECT
+from playwright.sync_api import expect
+import utils.gen3_admin_tasks as gat
+
+
+# TODO : enable this test after the manifest PRs are ready to roll
+@pytest.mark.wip
+class TestDataGuids:
+    variables = {}
+    dataguids_url = "https://dataguids.org"
+    non_existent_guids = [
+        "dg.ABCD/0000b4b4-2af4-42e2-9bfa-6fd11e5fb97a",
+        "0000b456-3r56-1dr3-0rt4-6fd11e5fb97a",
+    ]
+
+    @classmethod
+    def setup_class(cls):
+        cls.variables["guids"] = []
+        logger.info("Getting all the hosts from manifest.json ...")
+        host_list_path = TEST_DATA_PATH_OBJECT / "dataguids" / "host_list.txt"
```

Comment:
why are we relying on test data? we should be capturing these from the manifest

Reply:
so currently we do store the manifest.json in testdata folder, but it only has global block which is needed by graph submission tests. and it does not contain any other parts of manifest.json.